ci-operator/step-registry/cucushift/upgrade/helper-service/build-info/cucushift-upgrade-helper-service-build-info-commands.sh

```diff
@@ -9,6 +9,12 @@ BUILD_INFO_FILE="${ARTIFACT_DIR}/ocp-build-info"
 if [[ -n "${RELEASE_IMAGE_LATEST}" ]]; then
     echo "Release info for: ${RELEASE_IMAGE_LATEST}"
     oc adm release info "${RELEASE_IMAGE_LATEST}" || true
-    LATEST_VERSION="$(oc adm release info "${RELEASE_IMAGE_LATEST}" --output=jsonpath="{.metadata.version}")"
-    echo "${LATEST_VERSION}" > "${BUILD_INFO_FILE}"
+    INITIAL_VERSION="$(oc adm release info "${RELEASE_IMAGE_LATEST}" --output=jsonpath="{.metadata.version}")"
+    echo "INITIAL_VERSION : ${INITIAL_VERSION}" >> "${BUILD_INFO_FILE}"
```

Comment:
IMO, we should not add  the string `INITIAL_VERSION` in the file, it's hard for multi-version upgrade, e.g, we are required to add 4.3 -> 4.4 -> ... -> 4.12 upgrade

Reply:
fixed in new commit